audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/AuditMessage.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2019 Crown Copyright
```

Comment:
```suggestion
 * Copyright 2020 Crown Copyright
```
This will need to reviewed on the other AuditMessage files as well